src/app/pages/apps/components/installed-apps/app-settings-button/app-settings-button.component.html

```diff
@@ -32,6 +34,7 @@
     {{ 'Manage Container Images' | translate }}
   </button> -->
   <button
+    *ixRequiredRoles="manageCatalogRequiredRoles"
```

Comment:
This can be allowed, because there is no other way for user to see what trains are currently selected.
Related: https://github.com/truenas/webui/pull/10408